README.md

```diff
@@ -158,6 +158,20 @@ You can optionally create a test-coverage report in `coverage/*` like so:
 
     COVERAGE=yes bundle exec rake spec
 
+Feature tests run in headless Chrome. You can run them in an actual visible window by tagging the examples with `driver: :chrome`, like so:
```

Comment:
💟  This is great!

Can you toss in a "_for local debugging puposes_" so that we don't end up with committed `binding.pry`s? ;)

Reply:
Cool! I didn't know that 💃